app/views/works/_filters.html.erb

```diff
@@ -26,16 +26,16 @@
         </dt>
         <dd class="<%= filter_action %> tags group">
           <dl>
-            <% %w(rating warning category fandom character relationship freeform).each do |tag_type| %>
-              <dt id="toggle_<%= filter_action %>_<%= tag_type %>_tags" class="filter-toggle <%= tag_type %> tags">
+            <% %w(rating archive_warning category fandom character relationship freeform).each do |tag_type| %>
+              <dt id="toggle_<%= filter_action %>_<%= tag_type %>_tags" class="filter-toggle <%= tag_type_class(tag_type) %> tags">
                 <% # For accessibility, include filter action (e.g. Include) as landmark text %>
                 <% # The space needs to be in the landmark to keep the text aligned %>
                 <%= ts("<span class=\"landmark\">%{filter_action} </span>%{tag_type}",
                     filter_action: filter_action.titleize,
-                    tag_type: tag_type_label(tag_type)
+                    tag_type: tag_type_label_name(tag_type).pluralize
```

Comment:
Can we do the t thing here.

Reply:
It's used inside a `ts()`, so I don't think that's necessary?